README.md

```diff
@@ -24,6 +28,19 @@ MIT: http://rhardih.mit-license.org
 
 # Changelog
 
+**1.6**
+
+- fixed a few issues with the options page not displaying urls nicely
+- made the code smaller
+- fixed issue with ekill breaking pages like youtube or facebook
+- moved from replacing the whole HTML document to the built in remove()
+
+**1.5**
+
+- Added an options page 
+- made it possible to store elements a user has removed
+- added permission to access chrome.storage
+
```

Comment:
I think we'll just keep everything under a new 1.5 release, but I'll just fix that in the squash/merge process.